### ISSUE TITLE
Support WGLMakie

### ISSUE THREAD

**Issue by kbarros** [SunnySuite/Sunny.jl#46]
For Jupyter notebook use, it would be very nice to support WGLMakie in place of GLMakie. Currently blocked on this issue: https://github.com/MakieOrg/Makie.jl/issues/2243

**Comment by kbarros** [comment]
Maybe be possible now that https://github.com/MakieOrg/Makie.jl/issues/2243 has been resolved.

**Comment by kbarros** [comment]
Partial support has landed with https://github.com/SunnySuite/Sunny.jl/commit/370d8016c3ac6fcd8387e59b6b02a9a0583eaf8b. Now the user can select their Makie flavor by loading, e.g., `using GLMakie` or `using WGLMakie`, prior to calling the Sunny plotting functions.

WGLMakie support is still labeled "experimental" by the Makie developers. Problems I encountered:
 - The default figure size inside VSCode is really small, and there is no auto-resizing option: https://github.com/MakieOrg/Makie.jl/issues/1318.
 - Frequently, within VSCode, I get an error that the "socket timed out" when creating an `axis`. Running this command a second time usually works.

Nonetheless, it might be nice to have the option for WGLMakie, especially for people who are using remote clusters and are having trouble with the X-server connection.
